### PR TITLE
Phase 1

### PR DESCRIPTION
**Pull request by AhmedNasserG**
Task:
- [x] Add CourseCard component
- [x] Add CardsSlider component

### CODE REVIEW

File: src/components/CourseCard.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+					<p className="course-student-count">({props.course.studentCount})</p>
+				</div>
+				<p className="course-price">{props.course.price}</p>
+
+				{props.course.bestSeller ? (
+					<div className="best-seller">
+						{' '}
+						<p className="best-seller-text">Bestseller</p>
+					</div>
+				) : null}
```
Comment (AhmedMorgan) [inline src/components/CourseCard.js]:
```suggestion
				{props.course.bestSeller && (
					<div className="best-seller">
						{' '}
						<p className="best-seller-text">Bestseller</p>
					</div>
				)}
```

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,13 @@
 import logo from './logo.svg';
```
Comment (AhmedMorgan) [inline src/App.js]: remove any unused import
```suggestion
```

File: src/components/CourseCard.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+						starSpacing="1px"
+						starRatedColor="#E59719"
+					/>
+					<p className="course-student-count">({props.course.studentCount})</p>
+				</div>
+				<p className="course-price">{props.course.price}</p>
+
+				{props.course.bestSeller && (
+					<div className="best-seller">
+						{' '}
```
Comment (AhmedMorgan) [inline src/components/CourseCard.js]: why this line here?

Comment (AhmedNasserG) [reply src/components/CourseCard.js]: I was testing something and forgot to remove it, I will fix it now 👍

**Pull request merged**